docs/migration_guide.rst

```diff
@@ -1,6 +1,83 @@
 Migration guide
 ===============
 
+**********************
+0.18.2 Migration guide
+**********************
+
+Version 0.18.2 of **starknet.py** comes with support of RPC v0.4.0 Trace API!
+
+
+0.18.2 Breaking changes
+-----------------------
+
+.. currentmodule:: starknet_py.net.client
+
+1. :meth:`Client.get_block_traces` has been renamed to :meth:`Client.trace_block_transactions` in order to match RPC specification.
+
+
+0.18.2 Minor changes
+--------------------
+
+1. :meth:`Client.trace_block_transactions` return type has been changed from ``BlockTransactionTraces`` to ``Union[BlockTransactionTraces, List[BlockTransactionTrace]]``.
+
+.. currentmodule:: starknet_py.net.gateway_client
+
+2. Reverted error while using ``include_block`` in :meth:`GatewayClient.get_state_update` parameter on gateway mainnet.
+
+
+0.18.2 Development-related changes
+----------------------------------
+
+1. In order to be able to run tests, you must set some environmental variables:
+
+    - ``INTEGRATION_NODE_URL``
+    - ``TESTNET_NODE_URL``
+    - ``INTEGRATION_ACCOUNT_PRIVATE_KEY``
+    - ``INTEGRATION_ACCOUNT_ADDRESS``
+    - ``TESTNET_ACCOUNT_PRIVATE_KEY``
+    - ``TESTNET_ACCOUNT_ADDRESS``
+
+The best way to do that is to create ``test-variables.env`` file in ``starknet_py/tests/e2e/`` directory, so they can be loaded by the ``python-dotenv`` library.
+You can find an example file ``test-variables.env.template`` in the same directory with the format of how it should look like.
+
+
+|
+
+.. raw:: html
+
+  <hr>
+
+|
+
+**********************
+0.18.1 Migration guide
+**********************
+
+.. currentmodule:: starknet_py.net.gateway_client
+
+This version contains a quick fix to :meth:`GatewayClient.get_state_update` method (mainnet wasn't updated to 0.12.2 then).
+Additionally, accounts in Cairo1 are now supported!
```

Comment:
What does that mean?
edit; it'd be good to have the `cairo_version` change up here or sth, for it to be more visible